bin/kong

```diff
@@ -142,7 +142,7 @@ require("kong.cmd.init")("%s", %s)
 ]], cmd_name, args_str)
 
 local resty_cmd = string.format(
-  "resty --main-conf \"%s\" --http-conf \"%s\" --stream-conf \"%s\" -e '%s'",
+  "resty --shdict 'kong 5m' --shdict 'kong_locks 8m' --main-conf \"%s\" --http-conf \"%s\" --stream-conf \"%s\" -e '%s'",
```

Comment:
Why do we need these two `shdict` options?

Reply:
```
$ bin/kong check
resty --shdict 'kong 5m' --shdict 'kong_locks 8m' --main-conf "" --http-conf "" --stream-conf "" -e 'setmetatable(_G, nil)

pcall(require, "luarocks.loader")

package.path = (os.getenv("KONG_LUA_PATH_OVERRIDE") or "") .. "./?.lua;./?/init.lua;" .. package.path

require("kong.cmd.init")("check", { ["conf"] = "/etc/kong/kong.conf", ["v"] = false, ["vv"] = false, })
'
configuration at /etc/kong/kong.conf is valid
```

The pdk will require "kong" shared dict

```
ERROR: ./kong/pdk/node.lua:76: attempt to index local 'shm' (a nil value)
stack traceback:
        ./kong/pdk/node.lua:291: in function 'new'
        ./kong/pdk/init.lua:264: in function 'new'
        ./kong/global.lua:168: in function 'init_pdk'
        ./kong/cmd/start.lua:65: in function 'cmd_exec'
        ./kong/cmd/init.lua:31: in function <./kong/cmd/init.lua:15>
        (command line -e):7: in function 'inline_gen'
        init_worker_by_lua(nginx.conf:187):44: in function <init_worker_by_lua(nginx.conf:187):43>
        [C]: in function 'xpcall'
        init_worker_by_lua(nginx.conf:187):52: in function <init_worker_by_lua(nginx.conf:187):50>
```

Reply:
But I think that we dont need to use shared dict in pdk in this resty cmd, if we could resolve this problem, we maynot add kong/kong_locks here.

Reply:
hi @chronolaw , i revert this modification in bin/kong, now this PR only let bin/busted(test cases) disable the mocked ngx.shared.DICT APIs. The `bin/kong` command will still use the mocked one. Because if I remove these APIs, later developer may need to add more `shdict` options into `bin/kong` especially in kong-ee.

Reply:
Got. So I think that we should give this PR a more understandable titile.

Reply:
Ok, I rename it to "refactor(test): prevent use of mocked shared.DICT APIs in globalpatches.lua"